core/src/View/MDSpan/Kokkos_MDSpan_Accessor.hpp

```diff
@@ -442,7 +442,7 @@ class ReferenceCountedAccessor {
       data_handle_type p,
 #endif
       size_t i) const {
-    return data_handle_type(p, m_nested_acc.offset(p.get(), i));
+    return data_handle_type{p, m_nested_acc.offset(p.get(), i)};
```

Comment:
This one isn't buggy (it's a constructor call not a conversion) but I thought it might as well be included for consistency.